ReactiveSwift.xcodeproj/project.pbxproj

```diff
@@ -1278,7 +1278,7 @@
 				CURRENT_PROJECT_VERSION = 1;
 				ENABLE_TESTABILITY = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Can we update the deployment target of just the test target and leave the framework's deployment target at 10.9?

Reply:
I'll address that.

Reply:
Fixed in 8bdcc81.